src/gatling/simulations/uk/gov/hmcts/ccd/corecasedata/scenarios/manageCaseAssignments/casesharing.scala

```diff
@@ -0,0 +1,259 @@
+package uk.gov.hmcts.ccd.corecasedata.scenarios.manageCaseAssignments
+
+import com.typesafe.config._
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import uk.gov.hmcts.ccd.corecasedata.scenarios.utils.Environment
+
+object casesharing {
+
+  val config: Config = ConfigFactory.load()
+  val xuiClientId = "xuiwebapp"
+  val xuiScope = "openid profile roles manage-user create-user manage-roles search-user"
+  val ccdClientId = "ccd_gateway"
+  val ccdScope = "openid profile authorities acr roles openid profile roles"
+  val feedCSUserDataLarge = csv("CaseSharingUsers_Large.csv").circular
+  val feedCSUserDataSmall = csv("CaseSharingUsers_Small.csv").circular
+  val mcaUrl = "http://aac-manage-case-assignment-perftest.service.core-compute-perftest.internal"
```

Comment:
The perftest env shouldn't be hardcoded here, and this line might be better in the Environment file (with the others)

Reply:
I have now removed this file from the repo as the code is being rewritten as part of the manage case PI28